internal/host/installcmd_test.go

```diff
@@ -230,30 +230,16 @@ func postvalidation(isstepreplynil bool, issteperrnil bool, expectedstepreply *m
 	}
 }
 
-func validateInstallCommand(reply *models.Step, role models.HostRole, clusterId string, hostId string, hostname string) {
-	if hostname != "" {
-
-		installCommand := "podman run -v /dev:/dev:rw -v /opt:/opt:rw -v /run/systemd/journal/socket:/run/systemd/journal/socket " +
-			"--privileged --pid=host " +
-			"--net=host -v /var/log:/var/log:rw --env PULL_SECRET_TOKEN " +
-			"--name assisted-installer quay.io/ocpmetal/assisted-installer:latest --role %s " +
-			"--cluster-id %s " +
-			"--boot-device /dev/sdb --host-id %s --openshift-version 4.5 " +
-			"--controller-image %s --url %s --insecure=false --agent-image %s --host-name %s --installation-timeout %s"
-		ExpectWithOffset(1, reply.Args[1]).Should(Equal(fmt.Sprintf(installCommand, role, clusterId,
-			hostId, DefaultInstructionConfig.ControllerImage, DefaultInstructionConfig.ServiceBaseURL, DefaultInstructionConfig.InventoryImage, hostname,
-			strconv.Itoa(int(DefaultInstructionConfig.InstallationTimeout)))))
-	} else {
-		installCommand := "podman run -v /dev:/dev:rw -v /opt:/opt:rw -v /run/systemd/journal/socket:/run/systemd/journal/socket " +
-			"--privileged --pid=host " +
-			"--net=host -v /var/log:/var/log:rw --env PULL_SECRET_TOKEN " +
-			"--name assisted-installer quay.io/ocpmetal/assisted-installer:latest --role %s " +
-			"--cluster-id %s " +
-			"--boot-device /dev/sdb --host-id %s --openshift-version 4.5 " +
-			"--controller-image %s --url %s --insecure=false --agent-image %s --installation-timeout %s"
-		ExpectWithOffset(1, reply.Args[1]).Should(Equal(fmt.Sprintf(installCommand, role, clusterId,
-			hostId, DefaultInstructionConfig.ControllerImage, DefaultInstructionConfig.ServiceBaseURL,
-			DefaultInstructionConfig.InventoryImage, strconv.Itoa(int(DefaultInstructionConfig.InstallationTimeout)))))
-	}
+func validateInstallCommand(reply *models.Step, role models.HostRole, clusterId string, hostId string) {
+	installCommand := "podman run -v /dev:/dev:rw -v /opt:/opt:rw -v /run/systemd/journal/socket:/run/systemd/journal/socket " +
+		"--privileged --pid=host " +
+		"--net=host -v /var/log:/var/log:rw --env PULL_SECRET_TOKEN " +
+		"--name assisted-installer quay.io/ocpmetal/assisted-installer:latest --role %s " +
+		"--cluster-id %s " +
+		"--boot-device /dev/sdb --host-id %s --openshift-version 4.5 " +
+		"--controller-image %s --url %s --insecure=false --agent-image %s --installation-timeout %s"
```

Comment:
This has got me thinking about what kind of concerns we have for backward compatibility.
I would imagine we would never send this command and end up running an image that didn't include https://github.com/openshift/assisted-installer/pull/42, right?

The only thing I could think of was if we deployed this change to the service while someone was already running the installer image. They would still be able to download the master/worker ignition files as those APIs were left intact. Can you think of any other gotchas for this @avishayt @filanov ?

Reply:
If someone developing specific feature and want to test it then maybe he will use a different image name?

Reply:
Yeah, I was thinking more about in prod, not so much development.

Reply:
In production i think that the only thing that will change is the hash